Sources/_SwiftSyntaxMacros/ExpressionMacro.swift

```diff
@@ -12,7 +12,6 @@
 
 import SwiftSyntax
 import SwiftParser
```

Comment:
Looks like SwiftParser isn't needed here either

Reply:
You're right, the macro protocols have a bunch of extra imports. I've cleaned them up locally.